tests/integration/defs/disaggregated/test_ad_disagg.py

```
@@ -0,0 +1,1043 @@
+
+                futures = []
+                for request in requests:
+                    seed_disagg()
+                    try:
+                        result = llm.generate_async(
+                            request[0],
+                            sampling_params=request[1],
+                            disaggregated_params=request[2],
+                        )
+                        futures.append(result)
+                    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)